readthedocs/api/v2/templates/restapi/footer.html

```diff
@@ -69,9 +69,6 @@
         <dd>
           <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'builds_project_list' project.slug %}">{% trans "Builds" %}</a>
         </dd>
-        <dd>
-          <a href="//{{ settings.PRODUCTION_DOMAIN }}{% url 'project_downloads' project.slug %}">{% trans "Downloads" %}</a>
```

Comment:
I think we can continue to link to this, no? We still list the downloads on this page.

Reply:
From this comment (https://github.com/readthedocs/readthedocs.org/issues/5855#issuecomment-531351924) I understood we wanted to remove this link.

Reply:
From this comment (https://github.com/readthedocs/readthedocs.org/issues/5855#issuecomment-531351924) I understood we wanted to remove this link.

Reply:
I don't have a strong opinion here. I will re-add this link. We can always remove it later if we want.